radicle-cob/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "radicle-cob"
+version = "0.1.0"
+authors = ["Alex Good <alex@memoryandthought.me>", "Fintan Halpenny <fintan.halpenny@gmail.com>"]
+edition = "2018"
+license = "GPL-3.0-or-later"
+description = "Library for implementing Radicle Collaborative Objects"
+keywords = ["radicle", "collaborative objects", "cob", "cobs"]
+
+[lib]
+
+[dependencies]
+git-trailers = "0.1"
+petgraph = "0.5"
+regex = "1.5.5"
+serde_json = "1.0"
+thiserror = "1.0"
+toml = "0.5"
+tracing = "0.1"
+either = "1.6"
+multihash = "0.11"
+multibase = "0.9"
+lazy_static = "1.4"
+tempfile = "3.3"
```

Comment:
> `git-trailers` -> if this is just for signatures, we have a function in `storage::git::trailers` that does the job, we also have commit header (gpgsig) parsing in radicle-ssh

It's also needed to tag the author and "authority" in the commit though. As well as that, `radicle-cob` isn't depending on `radicle`, it'll be the other way around.

Happy to move to the gpgsig version, but wanted to get to a place where I had stuff working first :)

>  regex -> very curious why we need regex for a low level library such as this

It's used for `TypeName` validation.

>  toml -> can we use json, since we're already importing serde_json? our other docs (eg. identity) are all json

Sure, happy to do that. Maybe @alexjg can remind us why he opted for toml in the first place just to clarify first.

>  tracing -> not sure this is relevant without async, I think log should work

Yup, sounds good. Was planning on switching since I saw you used `log` everywhere else.

>  either -> probably not needed anymore, as it was used for direct/indirect delegations if I remember

You're right, it can be removed!

>  lazy_static -> we should replace with once_cell::Lazy

:+1: